vllm/config.py

```diff
@@ -3229,6 +3235,7 @@ class VllmConfig:
     additional_config: SupportsHash = field(default=None,
                                             init=True)  # type: ignore
     instance_id: str = ""
+    use_v1: bool = False
```

Comment:
`use_v1` should be added to at least 1 `compute_hash` function, no?

Reply:
Also, I don't understand how `use_v1` gets passed to child configs?

Reply:
Good point on the `compute_hash`

Ideally we dont have to propagate this down

Reply:
So how is e.g. `CacheConfig.use_v1` set? Separately from `VllmConfig.use_v1`?

Reply:
`VllmConfig` holds `CacheConfig`.

The reason that `CacheConfig` needs `use_v1` is because the `CacheConfig` is passed throgh attention layers while `VllmConfig` is not. I would prefer for only `VllmConfig` to have the `use_v1` argument

I hope to restructure the configs at some point in the future but that is out of scope for this PR